dxtbx/model/experiment_list.h

```diff
@@ -412,7 +412,8 @@ namespace dxtbx { namespace model {
       }
       scitbx::af::shared<std::size_t> result;
       for (std::size_t i = 0; i < size(); ++i) {
-         if (data_[i].get_profile() == obj || data_[i].get_imageset() == obj) {
+         if (data_[i].get_profile() == obj || data_[i].get_imageset() == obj ||
+           data_[i].get_scaling_model() == obj) {
```

Comment:
This might look nicer but I'm just being picky and it's only an opinion!
```
         if (data_[i].get_profile() == obj || 
             data_[i].get_imageset() == obj ||
             data_[i].get_scaling_model() == obj) {
```

Reply:
```python
if obj in (data_[i].get_profile(),
           data_[i].get_imageset(),
           data_[i].get_scaling_model()) {
```